src/utils/helpers/randomAddress.ts

```diff
@@ -1,4 +1,5 @@
 import * as Web3Utils from 'web3-utils';
 import { Address } from '~/utils/types';
 
-export const randomAddress = () => new Address(Web3Utils.randomHex(20));
+export const randomAddress = () =>
+  new Address(Web3Utils.randomHex(20)).toLowerCase();
```

Comment:
Why `toLowerCase`? I think its good practice to try to always use the check-summed addresses. But I also know that here and there that leads to problems (for example comparing).

Reply:
Changed it back